### PR TITLE
 Fix build warnings on DTS #833 

### PR DESCRIPTION
**Pull request by dancios**
Fixes Dasharo/dasharo-issues#833

### CODE REVIEW

File: meta-dts-distro/recipes-kernel/kernel/linux-yocto/efivars.cfg [hunk 1]
```diff
@@ -1 +1 @@
-CONFIG_EFI_VARS=y
```
Comment (dancios) [inline meta-dts-distro/recipes-kernel/kernel/linux-yocto/efivars.cfg:1]: resolves Dasharo/dts-scripts#29
This change is also requiere if this is sufficient

File: meta-dts-distro/recipes-gnome/gcab/gcab_%.bbappend [hunk 2]
```diff
@@ -0,0 +1 @@
+do_package_qa[noexec] = "1"
```
Comment (m-iwanicki) [inline meta-dts-distro/recipes-gnome/gcab/gcab_%.bbappend:1]: Can you use `insane_skip+=buildpaths` as in wolfssl so we don't need to disable whole qa check?

File: meta-dts-distro/recipes-kernel/kernel/linux-yocto_%.bbappend [hunk 3]
```diff
@@ -4,7 +4,6 @@ SRC_URI:append = " \
     file://acpi-bgrt.cfg \
     file://dts.cfg \
     file://enable-memconsole-coreboot.cfg \
-    file://efivars.cfg \
```
Comment (m-iwanicki) [inline meta-dts-distro/recipes-kernel/kernel/linux-yocto_%.bbappend:7]: Do we use this file or not? If not then delete file too if yes then this line probably should be here.

**Review by m-iwanicki — CHANGES_REQUESTED** [review]

**Comment by m-iwanicki** [comment]
Build completes successfully without warnings. 2 comments to be addressed and this PR can be merged.

File: meta-dts-distro/recipes-gnome/gcab/gcab_%.bbappend (hunk 2 above)
Comment (m-iwanicki) [reply meta-dts-distro/recipes-gnome/gcab/gcab_%.bbappend]: 647703793a3ae0b2f7bbc95a0f15d6407b6cff04

File: meta-dts-distro/recipes-kernel/kernel/linux-yocto_%.bbappend (hunk 3 above)
Comment (m-iwanicki) [reply meta-dts-distro/recipes-kernel/kernel/linux-yocto_%.bbappend:7]: 35d1227ef133879e42e9072991dc5d9616860f11

**Comment by m-iwanicki** [comment]
@PLangowski Minimal test on QEMU. It didn't work as QEMU can't be updated (yet).
Update started automatically.

```
Dasharo Tools Suite 2.0.0 DasharoToolsSuite ttyS0

DasharoToolsSuite login: root (automatic login)

Waiting for network connection ...
Network connection have been established!
Waiting for network connection ...
Network connection have been established!
Checking if board is Dasharo compatible.
Board vendor: Emulation is currently not supported
Gathering flash chip and chipset information...
Flash information: vendor="ST" name="M95M02"
Flash size: 0M
Firmware Update Mode detected; proceed with automatic update in 5 seconds
Updating in 5s:
5...
4...
3...
2...
1...
Waiting for system clock to be synced ...
Checking for the latest Dasharo update available...
Current Dasharo version: 0.2.0
Latest available Dasharo version:
ERROR Invalid version ''
Incorrect version format
Update cannot be performed
Starting bash session - please make sure you get logs from
/var/local/dts-err.log and /var/local/flashrom.log; then you can poweroff the platform
```

**Comment by m-iwanicki** [comment]
To test `Firmware Update Mode` with custom build

1. Enter BIOS -> Dasharo System Features -> Dasharo Security Options -> Enter Firmware Update Mode
2. Accept
3. Press correct key when asked
4. After `iPXE initialising devices... ok` finishes and you see `iPXE 1.21.1+ (g83861) -- Open Source Network Boot Firmware` message press CTRL+C once. After that enter `iPXE Shell` and boot custom DTS.

**Review by m-iwanicki — APPROVED** [review]

**Comment by artur-rs** [comment]
@PLangowski changes to be included in the v2.1.0 release

**Pull request merged**